bin/lwc-jest.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+'use strict';
+
+const { getArgs } = require('../src/utils/yargs');
+const { error } = require('../src/log');
+
+const args = getArgs();
+const runJest = require('../src/utils/test-runner');
```

Comment:
Can you move the all `require`s the top? Or is there any ordering issue?

Reply:
The original idea was that the user may pass in `--help` or some invalid args and then the `getArgs` call would terminate the app before needing to load the test runner.